cmd/controller-manager/app/server.go

```diff
@@ -128,8 +129,15 @@ func startControllers(config *rest.Config, opt *options.ServerOption) func(ctx c
 	controllerOpt.WorkerThreadsForPG = opt.WorkerThreadsForPG
 	controllerOpt.WorkerThreadsForGC = opt.WorkerThreadsForGC
 
+	checkControllersToStart(opt.Controllers)
```

Comment:
We print all the startable controllers here. @Monokaix

Reply:
Is this necessary? because we have print not enabled controllers in line 137, I mean we just print known controllers in ths flag: ）, previous implement is almost ok and just need print the known controllers registered in the flag.

Reply:
ok, so i will remove this